lib/utility.js

```diff
@@ -201,7 +201,7 @@ function counters () {
 function simd (simd) {
   return sharp.simd(is.bool(simd) ? simd : null);
 }
-simd(true);
+simd([0, "0", "false"].includes(process.env.VIPS_NOVECTOR) ? false : true);
```

Comment:
libvips checks for the presence rather than value of this env var, so we should probably do likewise (could use `is.defined()` to check?).

https://github.com/libvips/libvips/blob/e95a169a3c66e36c5062736f3c622aa06f924d02/libvips/iofuncs/vector.cpp#L89